From: Debian Go Packaging Team <pkg-go-maintainers@lists.alioth.debian.org>
Date: Fri, 12 Jul 2024 18:14:17 -0400
Subject: system-goimports

===================================================================
---
 pkg/bindings/generator/generator.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/bindings/generator/generator.go b/pkg/bindings/generator/generator.go
index 8f6234d..c74d505 100644
--- a/pkg/bindings/generator/generator.go
+++ b/pkg/bindings/generator/generator.go
@@ -169,7 +169,7 @@ func main() {
 		}
 
 		// go import file
-		goimport := exec.Command("../../../test/tools/build/goimports", "-w", out.Name())
+		goimport := exec.Command("goimports", "-w", out.Name())
 		goimport.Stderr = os.Stdout
 		if err := goimport.Run(); err != nil {
 			fmt.Println(err)
