From: Reinhard Tartler <siretart@tauware.de>
Date: Wed, 13 Nov 2024 21:51:47 -0500
Subject: TestUnitDirs: Skip when CLONE_NEWUSER fails

---
 cmd/quadlet/main_test.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cmd/quadlet/main_test.go b/cmd/quadlet/main_test.go
index 52bfcd1..6d24c8e 100644
--- a/cmd/quadlet/main_test.go
+++ b/cmd/quadlet/main_test.go
@@ -199,6 +199,9 @@ func TestUnitDirs(t *testing.T) {
 		}
 		c.Env = append(os.Environ(), "_UNSHARED=true")
 		err = c.Run()
+		if err != nil && strings.Contains(err.Error(), "operation not permitted") {
+			t.Skipf("Skipping test, CLONE_NEWUSER: %v", err)
+		}
 		assert.NoError(t, err)
 	} else {
 		fmt.Println(os.Args)
